lib/bootstrap.js

```diff
@@ -49,6 +51,15 @@ async function runCLI(cli, commandIsUsed) {
         cli.runHelp(process.argv);
         return;
     } else if (versionFlagExists) {
+        if (commandIsUsed && commandIsUsed.name)
+            try {
+                const pathForExternalPkg = path.resolve(process.cwd(), 'node_modules', '@webpack-cli', commandIsUsed.name);
+                const ExternalPkgJson = require(pathForExternalPkg + '/package.json');
+                const { name, version } = ExternalPkgJson;
+                process.stdout.write(name + ' ' + version);
+            } catch (e) {
+                process.stderr.write('ERR_EXTERNAL_PACKAGE_NOT_FOUND', e);
+            }
```

Comment:
Integrate this into `runHelp()` method of `WebpackCLI` class at `webpack-cli.js`. Would be even better if you integrate into `HelpGroup`.

P.S.
Can start by adding `command` as an argument to the function.